modules/rosa-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,49 @@
+:_module-type: PROCEDURE
+//Specify the _module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// rosa-nodes/rosa-autoscale-nodes.adoc
+
+[id="rosa-autoscaling_{context}"]
+= Autoscaling nodes in a cluster
+
+[role="_abstract"]
+Configure autoscaling to dynamically scale the number of worker nodes up or down based on load. Autoscaling for worker nodes is configured in the machine pool definition.
+
+Autoscaling is dependent on the resource quotas that are set. Verify resource quotas and request quota increases from the link:https://aws.amazon.com/console/[AWS console].
```

Comment:
Can we clarify the intent here?  The first time I read it it seemed like it was telling me that the autoscalers decision is based on the quotas set.  Maybe we can rephrase to "Successful autoscaling is dependent on having the correct resource quotas in your account".  Or something like that.

Reply:
Yeah +1.  The quota here is an AWS quota.  That is the distinction that should come across.

Reply:
Done.